packages/valory/connections/abci/tests/test_fuzz/test_fuzz.py

```diff
@@ -30,19 +32,25 @@
 )
 
 
-@pytest.mark.skip(reason="broken & takes too long time to complete on CI")
-class GrpcFuzzyTests(BaseFuzzyTests, TestCase):
+@pytest.mark.skipif(
+    platform.system() == "Windows",
+    reason="<IocpProactor overlapped#=1175 result#=0> is running after closing for ... seconds (windows_events.py:871)",
+)
+class TestFuzzyGrpc(BaseFuzzyTests):
     """Test the connection when gRPC is used"""
 
     CHANNEL_TYPE = GrpcChannel
     USE_GRPC = True
-    AGENT_TIMEOUT = 30  # 3 seconds
+    AGENT_TIMEOUT_SECONDS = 30
 
 
-@pytest.mark.skip(reason="broken & takes too long time to complete on CI")
-class TcpFuzzyTests(BaseFuzzyTests, TestCase):
+@pytest.mark.skipif(
+    platform.system() == "Windows",
+    reason="<IocpProactor overlapped#=1175 result#=0> is running after closing for ... seconds (windows_events.py:871)",
+)
+class TestFuzzyTcp(BaseFuzzyTests):
```

Comment:
we need it for test coverage, hence we only skip for Windows since this issue appears non-trivial to solve and not worth our effort at this point. Let me know if you want me to open an issue on this.

Reply:
the first hit when you search for "IocpProactor overlapped" results in what @solarw might consider leisure material
https://vstinner.github.io/asyncio-proactor-cancellation-from-hell.html